com.unity.ml-agents/Editor/BehaviorParametersEditor.cs

```diff
@@ -87,5 +111,33 @@ void DisplayFailedModelChecks()
                 }
             }
         }
+
+        void UpdateAgent(bool needModelUpdate, bool behaviorTypeChanged)
```

Comment:
nit: vague method name.  Perhaps `UpdateAgentBehavior` or `UpdateAgentBehaviorParameters`

Reply:
Changed to `UpdateAgentFromBehaviorParameters` (since the BehaviorParameters already get updated by the Editor)

Reply:
Renamed again to `UpdateAgentPolicy`